app/controllers/application_controller.rb

```diff
@@ -10,12 +10,22 @@ class ApplicationController < ActionController::Base
 
   before_action :authenticate_user!
 
+  rescue_from 'AccessGranted::AccessDenied' do
+    redirect_back_or_to(path: root_path, alert: 'Aktion nicht erlaubt!')
+  end
+
   def not_found
     raise ActionController::RoutingError, 'Not Found'
   end
 
   protected
 
+  def authorize_admin!
```

Comment:
IMAO this is bad naming. My first thought was you are making the user to an admin.